pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/AuxClassPathController.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+
+package net.sourceforge.pmd.util.fxdesigner;
+
+import java.io.File;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ResourceBundle;
+
+import net.sourceforge.pmd.util.fxdesigner.util.DesignerUtil;
+import net.sourceforge.pmd.util.fxdesigner.util.beans.SettingsOwner;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ListView;
+import javafx.stage.FileChooser;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+public class AuxClassPathController implements Initializable, SettingsOwner {
+
+    private final DesignerRoot designerRoot;
+    private List<File> list = new ArrayList<>();
+    @FXML
+    private Button removeFiles;
+    @FXML
+    private Button selectFile;
+    @FXML
+    private ListView<File> fileList;
+    @FXML
+    private final MainDesignerController parent;
+
+
+    public AuxClassPathController(DesignerRoot designerRoot, MainDesignerController parent) {
+        this.designerRoot = designerRoot;
+        this.parent = parent;
+    }
+
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+
+        selectFile.setOnAction(e -> fileSelected());
+        removeFiles.setOnAction(e -> setRemoveFiles());
+
+    }
+
+
+    private void fileSelected() {
```

Comment:
Usually the convention for methods that are executed on a specific event (handlers) is eg `onSelectFileClicked`, or `onRemoveFileClicked`. This makes it evident *when* the methods are executed.